kettleweb/static/style.css

```diff
@@ -29,23 +29,46 @@ body {
 }
 
 #header, #content, #footer, #flash {
-    width: 800px;
+    width: 1250px;
```

Comment:
I guess it doesn't hurt anything, depends on peoples opinions. Maybe this'll make mobile worse?